.golangci.yml

```diff
@@ -208,12 +203,12 @@ issues:
     - gocritic
     text: "deferInLoop: Possible resource leak, 'defer' is called in the 'for' loop"
     path: _test\.go
+  exclude-files:
+  - "zz_generated.*\\.go$"
 
 run:
```

Comment:
@Prajyot-Parab, regarding the message

> The configuration option `linters.staticcheck.go` is deprecated, please use global `run.go`.

Should the new equivalent be added in this section?

Reply:
@Amulyam24 on the basis of what I tested, not needed.

Reply:
checking something, hold on.

Reply:
fixed it. Thanks @Amulyam24 for pointing it out.